### ISSUE TITLE
Disengaging handbrake after TARDIS crash-land corrupts TARDIS dimension - unable to load into world 

### ISSUE THREAD

**Issue by wyvoe** [CommandrMoose/TardisRefined#355]
### Issue description

I've done this twice today with 2 different TARDISes, but after disengaging the handbrake _after_ a forced landing (crash-land, smack handbrake, etc.) the server AND singleplayer world (yes I've done it on both) both insta-crashed. 

The first time it happened, my TARDIS crash-landed someplace. In order to fly the TARDIS again, I had to interact with the handbrake. That's when the first crash happened. It corrupted the world in such a way that the server it was on was unable to start. When attempting to load the world in singleplayer, it still was not possible _until_ I fully deleted my TARDIS file in instance\saves\world\dimensions\tardis_refined.

After fully deleting my TARDIS dimension, I was able to load into the world without crashing to desktop. 
I tried to recreate the corruption in my friend's TARDIS; I succeeded after force-landing the TARDIS by smacking the handbrake, then disengaging the handbrake. Instant crash and corruption - unable to load back into the world at all, just like before.

TL;DR
Disengaging the TARDIS handbrake after a forced landing or crash landing corrupts the TARDIS dimension, making players unable to load into the world completely. Players are only able to load into the world if the corrupted TARDIS dimensions are deleted from the game files.

### Steps to reproduce

1. Force land TARDIS
2. Disengage handbrake
3. Crash
4. Discover that you are unable to load into your world and hopefully you've made a backup :]

### Minecraft version

1.20.1

### Mod Loader Platform type

Forge

### Mod Loader version

Forge 47.3.5

### Tardis Refined version

v2.0.6

### Attach the relevant crash report file or log file: (Don't know how to find a crash report file? See the [Minecraft Wiki Tutorial](https://minecraft.fandom.com/wiki/Tutorials/How_to_get_a_crash_report)

https://gist.github.com/wyvoe/69e3ce3551f6d3978d2f2dcbd3e70072

**Comment by wyvoe** [comment]
SO UPON EVEN FURTHER TESTING, this bug can only be recreated if one of the extra shells is equipped. The bug doesn't seem to be recreated when using a default TARDIS exterior shell.

**Comment by Jeryn99** [comment]
You have not updated extra shells...please try this and comment if it doesn't fix your issue
